docs/en_US/gsoc-ideas.md

```diff
@@ -0,0 +1,64 @@
+# GSoC 2024 Ideas List
+
+Hi there! This is the ideas list of OceanBase for Google Summer of Code 2024! 
+
+As a first-year mentor organization, we focus the development ideas on ob-operator, a kubernetes operator which helps deploy and manage OceanBase cluster on kubernetes cluster seamlessly. There are four projects for contributors, and we'll offer delicate guidance for every choice.
+
+Enjoy your summer of code!
+
+## 1. CLI tool
+
+* Project Description: The primary method to manage CRDs of ob-operator is manipulating YAML manifests which is not easy enough for new-coming users to get started. Within the CLI tool, features like component installation, demo setup,  resource management and necessary validation can be implemented.
```

Comment:
"Within the CLI tool, features like component installation, demo setup,  resource management and necessary validation can be implemented" this is a little bit strange
how about change it to 
"We'd like to provide features such as component installation, demo setup,  resource management and necessary validation within a CLI tool"